Backend/app/controllers/custom-form.controller.js

```diff
@@ -1,44 +1,69 @@
 const CustomForm = require('../models/custom-form.model');
-
+const logger = require('../logger');
 
 const customFormController = {
 
-    async getCustomForm(req, res) {
-        try {
-            const { form_id } = req.params;
-            const resForm = await CustomForm.queryForm({form_id: form_id});
-            res.send(resForm);
-        } catch (err) {
-            console.error(err);
-            res.status(500);
-            res.send({ error: err });
-        }
-    },
+  async getCustomForm(req, res, next) {
+    try {
+      const { form_id } = req.params;
+      logger.debug(`get custom form id ${form_id}`);
+      const resForm = await CustomForm.queryForm({ form_id: form_id });
+      res.send(resForm);
+    } catch (err) {
+      next(err);
+    }
+  },
+
+  async updateCustomForm(req, res, next) {
+    try {
+      const { form_id } = req.params;
+      const form = new CustomForm({ form_id: form_id, ...req.body });
+      logger.debug('update controller');
```

Comment:
Can you open a new issue to clean up the unused logger statements

Reply:
Okay, will do